source/install-java.rst

```diff
@@ -0,0 +1,34 @@
+安装 Java 运行环境
+==================
+
+:教程作者: `田冬冬 <https://me.seisman.info/>`__ (作者)、`姚家园 <https://github.com/core-man>`__ (审稿)
+:最近更新时间: 2020-12-31
+
+运行 Java 语言写的程序时，需要安装 Java 运行环境。
+
+在终端键入 ``java --version``，若显示版本信息，则表示 Java 运行环境已安装::
+
+    $ java --version
```

Comment:
I can use both `-version` and `--version` in macOS, but I can only use `-version` in CentOS.
```suggestion
    $ java -version
```

Reply:
OK, I believe it's only available in newer versions.